src/worker.js

```diff
@@ -12,10 +12,15 @@ var threadInfoStruct = 0; // Info area for this thread in Emscripten HEAP (share
 var selfThreadId = 0; // The ID of this thread. 0 if not hosting a pthread.
 var parentThreadId = 0; // The ID of the parent pthread that launched this thread.
 
-// Cannot use console.log or console.error in a web worker, since that would risk a browser deadlock! https://bugzilla.mozilla.org/show_bug.cgi?id=1049091
-// Therefore implement custom logging facility for threads running in a worker, which queue the messages to main thread to print.
 var Module = {};
 
+// If we use a custom export name, refer to Module from it as well, so that
+// we connect properly to the modularized instance which is the only thing
+// in the global scope.
+#if MODULARIZE_INSTANCE && EXPORT_NAME != 'Module'
+var {{{ EXPORT_NAME }}} = Module;
+#endif
```

Comment:
Would it work instead of 
```
var Module = {};


// If we use a custom export name, refer to Module from it as well, so that
// we connect properly to the modularized instance which is the only thing
// in the global scope.
#if MODULARIZE_INSTANCE && EXPORT_NAME != 'Module'
var {{{ EXPORT_NAME }}} = Module;
#endif
```
to do
```
var {{{ EXPORT_NAME }}} = {};
```
and in all places in the worker below, access `{{{ EXPORT_NAME }}}` instead of `Module`?

Reply:
That might work too, but I think it makes the code a little less readable, and there's a risk of future changes to the file forgetting and using `Module` (and then working except for the corner case fixed here).